salesforce-cdp/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -19,10 +19,18 @@ To use this connector, you must be familiar with:
 
 Before creating an app, you must have:
 
+* <<set-up-the-ingestion-api>>
```

Comment:
This should be in title caps. If you don't put a text here, it will be render as initial capped. Since it is in context, this is incorrect.